charts/gateway-api/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+name: gateway-api
+description: Helm Chart to install the Gateway API
+type: application
+version: 1.3.0
+appVersion: "1.3.0"
```

Comment:
at some point we should tie this appVersion with the Gateway release tag value, but for now this is good.